### DISCUSSION TITLE
Can i use NextAuth among multiple sessions?

### DISCUSSION THREAD

**Discussion by krishnanpandya007** [nextauthjs/next-auth#4427] [Help]
### Question 💬

Hello, currently i am working with nextauth and found amazing however if i login from tab A in browser, and then i am authenticated for tab A but when i go to tab B i need to re-login or re-siginIn again

Anyone knows how to deal with it??

### How to reproduce ☕️

```js
import CredentialsProvider from "next-auth/providers/credentials"

export default NextAuth({
  // Configure one or more authentication providers
  providers: [
    CredentialsProvider({

      name: "XYZ",

      credentials: {

        username: { label: "Username", type: 'text' },
        password: { label: "Password", type: 'password' },

      },

      async authorize(credentials, req) {
      
        const user = {id: 1, name: 'John Doe'}

        if (user) {

          return user;

        }else {

          return null

        }

      }
})

**Comment by nrsimha** [comment]
I came here to search for same thing, but then I realised that one tab I was opening as localhost:3000 and second tab where I signed in redirected me to 127.0.0.1:3000.
